containers/podman-desktop
# Start other informers lazily

### What does this PR do?

- start service informer when service store asks the first time the list of services
- stop the service informer on previous context when the current context changes

### What issues does this PR fix or reference?

Fixes partially #6202 

### How to test this PR?

Switch to the services list page and check that the services are visible
Debug logs indicate when the informer is started or not
Check by changing context
